src/main/resources/db/migration/changes/UsersAddMod.json

```
@@ -0,0 +1,36 @@
+{ "databaseChangeLog": [
+  {
+    "changeSet": {
+      "id": "UsersAddMod",
+      "author": "RiyaGupta",
```

```suggestion
      "id": "Users-001-AddMod",
      "author": "RiyaGupta1234",
```

Use your github id, not your real name.

Also: please change the name of this file from 
* Old: `src/main/resources/db/migration/changes/UsersAddMod.json`
* New: `src/main/resources/db/migration/changes/Users-001-AddMod.json`

---

The naming convention you have suggested causes the order of the migration files to be in the wrong order (Users-001-AddMod.json comes before Users.json instead of after). I have kept the current naming convention for now, please let me know if I should still change it to something else.

---

Alternatively, you could do something like Users001-AddMod.json.
Then in future migration files, it could continue with Users002-XYZ, etc

---

done!